SU-HSDO/suhumsci
# Mailto link color is default blue (incorrect)

# READY FOR REVIEW

## Summary
- Applies the proper link styles to all `mailto` type links regardless if they contain a `.mailto` class

## Need Review By (Date)
tbd

## Urgency
low

## Steps to Test
1. Pull code and setup the `economics` site (dark footer example), the `music` site (light footer example), and `islamicstudies` (traditional footer example).
2. View the footer in each site and verify the link styles are consistent for the mailto links
3. Go to https://music.suhumsci.loc/academics/degrees-and-other-studies/undergraduate-minor and verify the `Rowen Leigh` link in the body copy is also the correct link style.
